mediaqueries-4/Overview.bs

```diff
@@ -1695,6 +1695,160 @@ Color Display Quality: the 'color-gamut' feature</h3>
 	you can use this feature in a negated boolean-context fashion:
 	''not (color-gamut)''.
 
+<h3 id="dynamic-range">
+Dynamic Range: the 'dynamic-range' feature</h3>
+
+	<pre class='descdef mq'>
+	Name: dynamic-range
+	Value: standard | high
+	For: @media
+	Type: discrete
+	</pre>
+</h3>
+
+'@media/dynamic-range' represents the combination of brightness and color
+depth.
+
+<dl dfn-type=value dfn-for="@media/dynamic-range">
+	<dt><dfn>high</dfn>
+	<dd>
+		If the following are true for the UA and the output device:
+		* max brightness is considered high
+		* color depth is greater than 24 bit
+		* has a <a>high contrast ratio</a>
+	<dt><dfn>standard</dfn>
+	<dd>
+		If the following are true for the UA and the output device:
+		* max brightness is considered standard
+		* color depth is 24 bit or 8 bit per color component of RGB
+		* has a <a>low contrast ratio</a>
+</dl>
+
+<h3 id="contrast-brightness-of-display">
+	Determining contrast and brightness of display</h3>
+
+This text is non-normative. There currently is no agreed upon standardization of
+<dfn export>high contrast ratio</dfn>, <dfn export>low contrast ratio</dfn>,
+max brightness or low brightness as it relates to displays. As such, the
+determination for '@media/dynamic-range' and '@media/video-dynamic-range' will
+be defined by the user agent.
+
+<h3 id="video-prefixed-properties">
+Video Prefixed Properties scenarios</h3>
```

Comment:
Sorry, let me revise my advise. I'd name this "Video Prefixed Feature", not properties, and no mention of scenarios. "Features" is more correct from the CSS pov.